pytest.ini

```diff
@@ -1,4 +1,4 @@
 [pytest]
 addopts = --disable-socket
 cache_dir = build/pytest
-python_files = test/test_*.py
+python_files = test_*.py
```

Comment:
since this is Python 3.5, this could probably be `test/**/test_*.py`

Reply:
https://stackoverflow.com/questions/2186525/use-a-glob-to-find-files-recursively-in-python

Reply:
I tried that first, but Pytest uses `path.fnmatch` rather than `glob.glob`, so it only results in the contents of 1 directory deep being found. Planning to open a PR.